tests/functional/dashboard/catalogue_tests.py

```diff
@@ -75,14 +75,14 @@ def test_upc_filter(self):
         self.assertNotIn(product3, products_on_page)
 
 
-class TestAStaffUser(WebTestCase):
+class TestAStaffUserEdit(WebTestCase):
```

Comment:
I had to create a separate TestCase or this test failed strangely and inconsistently.